docs/source/cheat_sheet_py3.rst

```diff
@@ -122,6 +122,13 @@ When you're puzzled or when things are complicated
    # dynamic to write a type for.
    x = mystery_function()  # type: Any
 
+   # This is how to deal with varargs.
+   # This makes each positional arg and each keyword arg a 'str'.
+   def call(self, *args, **kwargs):
+            # type: (*str, **str) -> str
```

Comment:
That had occurred to me too after pushing this, and through some fat-fingering I didn't push the fix. Now I have.